cl/alerts/utils.py

```diff
@@ -726,3 +729,21 @@ def prepare_percolator_content(app_label: str, document_id: str) -> tuple[
             )
 
     return percolator_index, es_document_index, documents_to_percolate
+
+
+def build_alert_email_subject(hits: list[SearchAlertHitType]) -> str:
+    """Build the email subject line for search alert emails.
+
+    "X Alert(s) have hits: alert_name_1, alert_name_2 ..."
+
+    :param hits: A list of search alert hits, where each hit contains the
+    alert details.
+    :return: A string representing the email subject line.
+    """
+
+    alerts_count = len(hits)
+    alert_names_str = ", ".join(alert_hit[0].name for alert_hit in hits)
+    alert_subject = f"{alerts_count} Alert{pluralize(alerts_count)} have hits: {alert_names_str}"
```

Comment:
I played with using pluralize, but I think it might just be easier to do it this way:

```suggestion
    if alert_count > 1:
        alert_subject = f"{alerts_count} alerts have hits: {alert_names_str}"
    else:
        alert_subject = f"{alerts_count} alert has hits: {alert_names_str}"
```

But pluralize works too! :)

Reply:
Sure, this is simpler.